Misc/reputations.json

```diff
@@ -6,7 +6,7 @@
       "version": -1,
       "regex": "",
       "details": "File enhancement scripts",
-      "enhancementScriptNames": ["Strings", "CYFileRep"]
+      "enhancementScriptNames": ["Strings", "CYFileRep", "SandboxDetonateFile", "ParseEmailFile"]
```

Comment:
Did you test you can execute these scripts  "SandboxDetonateFile", "ParseEmailFile" from reuptation pop up screen?